docs/Glossary/index.md

```diff
@@ -0,0 +1,143 @@
+# Glossary
+
+This glossary provides definitions and explanations of commonly used terms, abbreviations, and phrases organized by the relevant application or function.
+
+---
+
+## Media Management Applications
+
+**Radarr**, **Sonarr**, **Lidarr**, and **Prowlarr** are applications designed to help with management of your media libraries and automation. This group of applications is commonly referenced collectively as the "arrs".
+
+| Application                          | Content Type       |
+| ------------------------------------ | ------------------ |
+| [**Radarr**](../Radarr/index.md)     | Movies             |
+| [**Sonarr**](../Sonarr/index.md)     | TV Shows/Series    |
+| [**Lidarr**](../Lidarr/index.md)     | Music              |
+| [**Prowlarr**](../Prowlarr/index.md) | Indexer Management |
+
+### Custom Format
+
+A feature that uses pattern matching on a release, based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile). After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.
+
+### Quality
+
+Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.
+
+### Quality Definitions
+
+Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+A defined group and preferential ordering of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
+
+---
+
+## Torrents
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process.
```

Comment:
```suggestion
The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process. `Cross-Seed` and `crossseed` or `xseed` often refer to the application.  While `cross seed` or `xseed` can refer to the action. Context dependent.
```

Reply:
technically context-dependent is hyphenated according to Grammarly :P

Reply:
Not sure we need 'context dependent', so removed, and make one more tweak. Good to get the extra terms in there though.